common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -212,14 +212,16 @@ object MetricEmitter {
 
   val metrics = Kamon.metrics
 
-  def emitCounterMetric(token: LogMarkerToken): Unit = {
+  def emitCounterMetric(token: LogMarkerToken): Unit = emitCounterMetric(token, 1)
+
+  def emitCounterMetric(token: LogMarkerToken, times: Long): Unit = {
```

Comment:
Can this have a default value for `time` instead of overloading?

Reply:
Makes sense. Was doing it the traditional java way here!